quickstart/nodejs/README.md

```diff
@@ -26,8 +26,8 @@ For examples of using other Node.js clients, see the individual documentation fo
 Add environment variables for your cache's **HOST NAME** and **Primary** access key. You will use these variables from your code instead of including the sensitive information directly in your code.
 
 ```
-set REDISCACHEHOSTNAME=contosoCache.redis.cache.windows.net
-set REDISCACHEKEY=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+set AZURE_CACHE_FOR_REDIS_RESOURCE_NAME=contosoCache
```

Comment:
I disagree with this, as it adds a layer of understanding the customer may not have at this point when they are new to Azure.

Reply:
See the Portal experience for Redis - all information about connecting to a cache always includes full host name

Reply:
Variable name should be "host name" rather than "resource name", and example value should include the full host name: contosoCache.redis.cache.windows.net